spring-data-jdbc/src/test/java/org/springframework/data/jdbc/core/JdbcAggregateTemplateIntegrationTests.java

```diff
@@ -466,7 +461,38 @@ public void saveAndLoadLongChain() {
 		template.delete(chain4, Chain4.class);
 
 		String countSelect = "SELECT COUNT(*) FROM %s";
-		jdbcTemplate.queryForObject(String.format(countSelect, "CHAIN0"),emptyMap(), Long.class);
+		jdbcTemplate.queryForObject(String.format(countSelect, "CHAIN0"), emptyMap(), Long.class);
```

Comment:
Why String.format(…) instead of having directly SELECT COUNT(*) CHAIN0? Also, the query result is not used/asserted.